clients/drcachesim/reader/config_reader.cpp

```diff
@@ -143,6 +143,18 @@ config_reader_t::configure(const std::string &config_file, cache_simulator_knobs
                 ERRMSG("Error reading verbose from the configuration file\n");
                 return false;
             }
+        } else if (param == "coherence") {
```

Comment:
I added the parameter to the supported configuration file options in that file. Should I document it further in that file?

Reply:
Yes, please remove the Current Limitations line about coherence.  The option docs will be auto-filled-in.

I think this should probably also be listed in the top-level changelog list of features at the bottom of this list as a new drcachesim feature of interest to users: https://github.com/DynamoRIO/dynamorio/blob/master/api/docs/release.dox#L203

Reply:
Perhaps this should also be on by default.   Is your plan to make it default-on in a separate PR?

Reply:
I was hesitant to turn this on by default, as it would change the current expected outputs from the simulator, fail tests, and change results for anyone already using the simulator. I think it makes sense to be enabled by default, and if you agree I could implement this in a separate PR. Alternatively, if you think it makes sense to make it default now, I could just switch the flag to a no_coherence and invert the logic. Do you think such a change should be included in this PR or implemented in a separate PR?

Reply:
I would do it separately.  Like you said, it could cause instability, and we (and others) can more easily identify and revert a tiny flag-switch PR.